MakeProject/README

```diff
@@ -0,0 +1,20 @@
+A cmake file that builds and links two files.
+
+Used files:
+
+CMakeLists.txt - the actual cmake file
+include - fun.h
+source - fun.cpp, main.cpp
+
+Usage:
+
+1) run "cmake -S <path1> -B <path2>" command
```

Comment:
Make it simple. when writing Readme. You must know how your project is constructed: What directories and files you have and where are they located. Based on that knowledge you must write readme. You must not think that others will move your files/directories. If they do so, then it's their problem to fix build command

Reply:
Here must be written a command, which I can copy-paste and it would work